src/main/kotlin/no/nav/helse/fritakagp/auth/AuthClient.kt

```diff
@@ -75,7 +74,7 @@ class AuthClient(
         TokenResponse.Error(e.response.body<TokenErrorResponse>(), e.response.status)
     }
 
-    suspend fun exchange(target: String, userToken: String, provider: IdentityProvider): TokenResponse = try {
+    suspend fun exchange(provider: IdentityProvider, target: String, userToken: String): TokenResponse = try {
```

Comment:
tenker det er best hvis rekkefølgen er lik for på argumenter i token, exchange og introspect